### ISSUE TITLE
Automatic Spatially-aware Fashion Concept Discovery

### ISSUE THREAD

**Issue by jchanxtarov** [jchanxtarov/arXivNotes#31]
# Basic Information
- Authors: Xintong Han, Zuxuan Wu, Phoenix Huang, Xiao Zhang
- Date: Aug, 2017
- Published By: arXiv

## Link
https://arxiv.org/pdf/1708.01311.pdf

# Overview
![image](https://user-images.githubusercontent.com/41275034/141017616-38162d35-3dd0-4b99-ab19-7a50aa8ee941.png)
- アイテム画像と単語（商品説明）を同一空間に埋め込む（image(pink skirt) - pint + blue = blue skirt）
- カテゴリごとに個別の空間に埋め込むので，モデルは個別に学習
- Stochastic bidirectional contrastive loss (to achieve good convergence)
   - contrastive loss: min( max( 0, hyperparam - cosine_dis(pos_img_embed, pos_rev_embed) + cosine_dis(pos_img_embed, neg_rev_embed) ) + max( 0, hyperparam - cosine_dis(pos_rev_embed, pos_img_embed) + cosine_dis(pos_rev_embed, neg_img_embed) ) )
- pre-trained CNN(GoogleNet, GAP layer) -> 2048
 ...
(面白いので別途解説を書く)

# Others

# Reference (for understanding)

**Comment by jchanxtarov** [comment]
実装する
